app/LoginPage/page.js

```diff
@@ -0,0 +1,82 @@
+"use client";
```

Comment:
[Question]
What's this for?

Reply:
This is used for Next.js client side rendering....i was required to add it because i made use of some react hooks on the page